hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -426,7 +428,7 @@ func (r *NodePoolReconciler) reconcile(ctx context.Context, hcluster *hyperv1.Ho
 		}
 		removeStatusCondition(&nodePool.Status.Conditions, hyperv1.NodePoolValidMachineConfigConditionType)
 
-		kubevirtBootImage, err = kubevirt.GetImage(nodePool, releaseImage)
+		kubevirtBootImage, err = kubevirt.GetImage(nodePool, releaseImage, controlPlaneNamespace)
```

Comment:
to get a head start on caching, maybe we can move this up in the reconcile loop to before we wait for the ignition server to come online. That would let us start caching even before the kubevirtmachines are created and the capk controller is online